trex-scripts/tests/port_shaping_rate.py

```diff
@@ -19,6 +19,7 @@
 
 SENDER_PORT   = [0]
 RECEIVER_PORT = [1]
+Shaping_rate  = 1000 # in Mbps
```

Comment:
The convention for constants is to use upper case. Also adding the unit to the name usually improves readability.
```suggestion
SHAPING_RATE_MBPS  = 1000 # in Mbps
```